Build errors when using `default-features = false`.

When using `default-features = false` in Cargo.toml such as:

```
source-span = {version= "2.2.0", default-features = false}
```

I get the following build errors:

```
error[E0282]: type annotations needed for `std::option::Option<T>`
   --> C:\Users\Byte\.cargo\registry\src\github.com-1ecc6299db9ec823\source-span-2.2.0\src\fmt.rs:807:27
    |
807 |         let mut current_color = None;
    |             -----------------   ^^^^ cannot infer type for type parameter `T` declared on the enum `Option`
    |             |
    |             consider giving `current_color` the explicit type `std::option::Option<T>`, where the type parameter `T` is specified

error[E0599]: no associated item named `Blue` found for unit type `()` in the current scope
    --> C:\Users\Byte\.cargo\registry\src\github.com-1ecc6299db9ec823\source-span-2.2.0\src\fmt.rs:1194:25
     |
1194 |             margin_color: Color::Blue,
     |                                  ^^^^ associated item not found in `()`

error: aborting due to 2 previous errors
```

--- a/src/fmt.rs
+++ b/src/fmt.rs
@@ -804,6 +804,7 @@ impl CharMap {
 
 impl fmt::Display for CharMap {
 	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		#[cfg(feature = "colors")]
 		let mut current_color = None;
 		for y in 0..self.height {
 			for x in 0..self.width {
@@ -1191,6 +1192,9 @@ impl Default for Formatter {
 	fn default() -> Formatter {
 		Formatter {
 			highlights: Vec::new(),
+			#[cfg(not(feature = "colors"))]
+			margin_color: (),
+			#[cfg(feature = "colors")]
 			margin_color: Color::Blue,
 			viewbox: Some(2),
 			show_line_numbers: true,

